moveit_ros/move_group/src/default_capabilities/kinematics_service_capability.cpp

```diff
@@ -145,6 +145,10 @@ void MoveGroupKinematicsService::computeIK(moveit_msgs::PositionIKRequest& req,
 bool MoveGroupKinematicsService::computeIKService(moveit_msgs::GetPositionIK::Request& req,
                                                   moveit_msgs::GetPositionIK::Response& res)
 {
+  if (req.ik_request.robot_state.joint_state.position.empty())
+  {
+    ROS_WARN("Empty joint_state position in GetPositionIK::Request msgs. Starting from the current robot state");
```

Comment:
```suggestion
    ROS_WARN_NAMED(getName(), "Empty joint_state position in GetPositionIK::Request msgs. Starting from the current robot state");
```
This way messages can be filtered by category and verbosity be selectively reduced.